docs/operating-systems/course-tools.md

```diff
@@ -201,70 +197,94 @@ You want your output to say you have the clang version.
           gcc -c hello.c
           
       clean:
-          rm hello.o hello.exe```
-9. Navigate to the directory in your terminal, enter make and see the program run!
+          rm hello.o hello.exe
+      ```
+ 
+Navigate to the directory in your terminal, and run the make command.
 
 ### Docker
-- Navigate to https://docs.docker.com/desktop/install/windows-install/, get the installer, and follow the instructions.
+Navigate to https://docs.docker.com/desktop/install/windows-install/ to install Docker.
 
 ### Python 3.10
-1. Go to the Microsoft Store app, search ```Python 3.10``` and install.
-2. Can check this worked by using the ```python --version``` command.
-
-### Golang
-1. Download Go from the official website (https://go.dev/dl/).
-2. Follow the installer's instructions and wait while it is downloaded onto your laptop.
-3. Search for ```env``` on your computer and open environment variables.
-4. Copy the path for Go and paste it into your user variable editing window.
-5. Test that go is working by typing ```go version``` into your terminal.
-
-### pipx / poetry / gatorgrade
+Go to the Microsoft Store app to install: 
+```
+Python 3.10
+```
 
-1. install pipx using the command ```python3 -m pip install --user pipx``` and then ```python3 -m pipx ensurepath```.
 
-2. you can then install gatorgrade using pipx using the command ```pipx install gatorgrade```.
+Check version:
+```
+python --version
+``` 
+ 
 
-3. Use the ```curl -sSL https://install.python-poetry.org | python3 -``` command to install poetry.
+### Golang
+- Download Go from the official website (https://go.dev/dl/).
```

Comment:
I would recommend you change this link to the format you used above in the different languages.